src/components/DeleteConfirmation/DeleteConfirmation.scss

```diff
@@ -1 +1,44 @@
 @import "../../pages/Index/index.scss";
+
+.confirmation-background {
+    background-color: white;
+    border: 4px solid $assignments-font-color;
+    box-shadow: "5px 10px #888888";
```

Comment:
are the shadows all similar enough to standardize into a variable?

Reply:
I think this one is only used twice